src/github/deployment.ts

```diff
@@ -1,30 +1,99 @@
 import transformDeployment from "../transforms/deployment";
 import { emitWebhookProcessedMetrics } from "../util/webhooks";
 import { CustomContext } from "./middleware";
-import { DeploymentsResult } from "../jira/client";
+import getJiraClient, { DeploymentsResult } from "../jira/client";
+import { booleanFlag, BooleanFlags } from "../config/feature-flags";
+import sqsQueues from "../sqs/queues";
+import { GitHubAPI } from "probot";
+import { WebhookPayloadDeploymentStatus } from "@octokit/webhooks";
+import { LoggerWithTarget } from "probot/lib/wrap-logger";
 
-export default async (context: CustomContext, jiraClient): Promise<void> => {
-	const jiraPayload = await transformDeployment(context.github, context.payload, jiraClient.baseURL, context.log);
+export default async (context: CustomContext, jiraClient, _util, githubInstallationId: number): Promise<void> => {
```

Comment:
If we're not using util, just remove it?

Reply:
it is part of the "interface" that is used in the other webhook functions, so I couldn't just remove that param here